rudder-core/src/main/scala/com/normation/rudder/services/nodes/NodeInfoService.scala

```diff
@@ -83,7 +116,7 @@ trait NodeInfoService {
    * @param nodeId
    * @return
    */
-  def getNodeInfo(nodeId: NodeId) : Box[NodeInfo]
+  def getNodeInfo(nodeId: NodeId) : Box[Option[NodeInfo]]
```

Comment:
To me, it's adding lot of boilerplate handling of the None case and emptybox case to do the same thing with specifying an error message for None